### ISSUE TITLE
Ionic 4 Angular 8 - Android app crashes  depending on the pdf file

### ISSUE THREAD

**Issue by BemHur** [intbot/ng2-pdfjs-viewer#83]
Hi there,
Im trying to use this lib with a Ionic 4 app and it works nice, but ti tryed with this pdf:
https://argus-zenite.sfo2.digitaloceanspaces.com/simulado/DF0C08E8B1445FAAD4D3C7283FAACF06E50ACD9EC77E37F3DC3B09E191E5963A.pdf
and the app crashes when a roll out to the middle/end of the pages.

Can someone help, any suggestions are welcome.

**Comment by Ng-India99** [comment]
Did you got the solution ?

**Comment by BemHur** [comment]
No, i ended up not using this lib and opening the pdf with the native pdf app
